roles/core/conman/molecule/default/verify.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Verify
+  hosts: management1
```

Comment:
Prefer 'all' unless this is specific to a single host out of a group.

Reply:
This test suite is executed only against the management.
As a compute is declared in `molecule.yml` if we declare `hosts: all` the tests will try to be executed also against the compute and the tests will fail.

Reply:
Please don't change molecule.yml, instance is fine for the test.